tests/Fixer/FunctionNotation/FunctionDeclarationFixerTest.php

```diff
@@ -161,6 +171,141 @@ function foo() /* bar */
      * @param string      $expected
      * @param null|string $input
      *
+     * @dataProvider provideClosureFunctionSpacingNoneCases
+     */
+    public function testFixClosureFunctionSpacingNone($expected, $input = null)
```

Comment:
> `$expected === $input`

input is nullable, so 
`[ $expected, null, null ]` (which is the same as ` [ $expected ]`
or
`[ $expected, null, $config ]`

-----------------------

strongly disagree, with later it way easier to add more cases (without need to change return statement evaluation) an we could use custom labels, like
`return [
    'no changes' => [$expected],
    'changes' => [$expected, $input],
    'changes with one space' => [$expected, $input, $config],
    'changes without space' => [$expected, $input, $config],
]'`